index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <title>My_portfolio</title>
+</head>
+<body>
+    <header class="headerA">
+        <div>
+            <p class="logo">Bertin</p>
+        </div>
+        <div>
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+        </div>
```

Comment:
- [ ]  Please wrap the three bars in a `nav` tag. The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.
` <nav>`
            ` <div class="bar1"></div>`
            ` <div class="bar2"></div>`
           `  <div class="bar3"></div>`
  ` </nav>`